README.md

```diff
@@ -15,11 +15,27 @@ Installation
 Just add the following line to `project/plugins.sbt`:
 
 ```
-addSbtPlugin("com.databricks" %% "sbt-databricks" % "0.1.1")
+addSbtPlugin("com.databricks" %% "sbt-databricks" % "0.1.2")
 ```
 
-Usage - Deployment
-==================
+#### Enable sbt-databricks for all your projects
+
+`sbt-databricks` can be enabled as a [global plugin](http://www.scala-sbt.org/0.13/tutorial/Using-Plugins.html#Global+plugins)
+ for use in all of your projects in two easy steps:
+
+1. Add the following line to `~/.sbt/0.13/plugins/build.sbt`:
+
+```
+addSbtPlugin("com.databricks" %% "sbt-databricks" % "0.1.2")
+```
+
+2. Set the settings defined [here](#settings) in `~/.sbt/0.13/databricks.sbt`.
```

Comment:
I think in the rendered version this number appears as 1 instead of 2. I think the markdown is really stupid when there is a code block in between a numbered list